target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -80,6 +80,11 @@ comfast,cf-e560ac)
 	ucidef_set_led_switch "lan3" "LAN3" "$boardname:blue:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "$boardname:blue:lan4" "switch0" "0x10"
 	;;
+comfast,cf-wr752ac)
+        ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
```

Comment:
Keep this line, as trigger won't work here, and remove the WLAN lines, as this is done by trigger.
Please check whether you can merge with another case then.